fiftyone/operators/store/service.py

```
@@ -16,14 +16,20 @@
 class ExecutionStoreService:
     """Service for managing execution store operations."""
 
-    def __init__(self, repo: Optional["ExecutionStoreRepo"] = None):
+    def __init__(
+        self,
+        repo: Optional["ExecutionStoreRepo"] = None,
```

Ruff

Undefined name `ExecutionStoreRepo`

(F821)